src/main/java/frc/robot/OperatorInterface.java

```diff
@@ -20,89 +17,82 @@
+                .whenReleased(commandFactory.stopIntake())
                 .add();
         
         operatorPanelManager.addButton(RobotConstants.BUTTONS.HOOD_FORWARD_ADJUST)
-                .whileHeld(new AdjustRaiseHoodCommand(subsystemManager.getShooterSubsystem()))
+                .whileHeld(commandFactory.nudgeHoodForward())
                 .add();
         
         operatorPanelManager.addButton(RobotConstants.BUTTONS.HOOD_BACKWARD_ADJUST)
-                .whileHeld(new AdjustBackwardHoodCommand(subsystemManager.getShooterSubsystem()))
+                .whileHeld(commandFactory.nudgeHoodBackward())
                 .add();
                 
-        operatorPanelManager.addButton(RobotConstants.BUTTONS.SELECT_PRESET_1)
-                .whenPressed(subsystemManager.getShooterSubsystem().selectPreset1())
-                .add();
+        //operatorPanelManager.addButton(RobotConstants.BUTTONS.SELECT_PRESET_1)
+        //        .whenPressed(subsystemManager.getShooterSubsystem().selectPreset1())
+        //        .add();
         
-        operatorPanelManager.addButton(RobotConstants.BUTTONS.SELECT_PRESET_2)
-                .whenPressed(subsystemManager.getShooterSubsystem().selectPreset2())
-                .add();
+        //operatorPanelManager.addButton(RobotConstants.BUTTONS.SELECT_PRESET_2)
+        //        .whenPressed(subsystemManager.getShooterSubsystem().selectPreset2())
+        //        .add();
         
         drive = subsystemManager.getDriveSubsystem();
         
         joystickManager.addButton(RobotConstants.BUTTONS.SNAP_TO_TARGET)
-                .whenPressed(new SnapToVisionTargetCommand(drive).withTimeout(5))
+                .whenPressed(commandFactory.snapToVisionTargetCommand().withTimeout(5))
                 .add();
         
         joystickManager.addButton(RobotConstants.BUTTONS.DRIVER_SHOOT)
-                .whenPressed(subsystemManager.getIntakeSubsystem().startElevator())
-                .whenReleased(subsystemManager.getIntakeSubsystem().stopElevator())
+                .whileHeld(commandFactory.fireCommand())
                 .add();
         
         joystickManager.addButton(12)
-                .whenPressed(new HoodHomingCommand(subsystemManager.getShooterSubsystem()))
+                .whenPressed(commandFactory.hoodHomeCommand())
                 .add();
-        
-//        joystickManager.addButton(9)
-//                .whenPressed(new DriveForwardSetDistance(drive, 24))
-//                .add();
-//        
-//        joystickManager.addButton(8)
-//                .whenPressed(new SnapToYawCommand(drive, 90, true))
-//                .add();
+
         
         joystickManager.addButton(RobotConstants.BUTTONS.OUTAKE)
-                .whenPressed(subsystemManager.getIntakeSubsystem().reverse())
-                .whenReleased(subsystemManager.getIntakeSubsystem().stopEverything())
+                .whenPressed(commandFactory.reverse())
```

Comment:
We should use something a little more descriptive then reverse.

Reply:
i agree-- i just ported what was there. i'm quite sure a lot of these may not even be needed